.eslintrc

```diff
@@ -20,16 +16,18 @@
     "ecmaVersion": 11,
     "sourceType": "module"
   },
-  "plugins": ["react", "jest"],
+  "plugins": ["react", "jest", "react-hooks"],
   "rules": {
-    "indent": ["error", 2],
-    "linebreak-style": ["error", "unix"],
     "quotes": ["error", "single"],
```

Comment:
I'm concerned about this many linting rules being turned off.  Many of these provide valuable guards against mistakes and can be handled as inline `eslint-disables` where the rule really is counterproductive.